integration/travis/run_integration.sh

```diff
@@ -163,9 +186,14 @@ export COOK_SLAVE_URL=http://localhost:12323
 export COOK_MESOS_LEADER_URL=${MINIMESOS_MASTER}
 {
     echo "Using Mesos leader URL: ${COOK_MESOS_LEADER_URL}"
-    pytest -n4 -v --color=no --timeout-method=thread --boxed -m "not serial" || test_failures=true
-    pytest -n0 -v --color=no --timeout-method=thread --boxed -m "serial" || test_failures=true
+    if [ "$JOB_LAUNCH_RATE_LIMIT" = off ]; then 
+      pytest -n4 -v --color=no --timeout-method=thread --boxed -m "not serial" || test_failures=true
+      pytest -n0 -v --color=no --timeout-method=thread --boxed -m "serial" || test_failures=true
+    else
+      pytest -n0 -v --color=no --timeout-method=thread --boxed -m multi_user tests/cook/test_multi_user.py -k test_rate_limit_launching_jobs || test_failures=true
+    fi
```

Comment:
It would be nice to add a comment somewhere explaining why we are isolating this test from the other tests

Reply:
Done in the .travis.yml file.